include/solana.hpp

```diff
@@ -554,6 +579,23 @@ class Connection {
    **/
   Version getVersion() const;
 
+  uint64_t getminimumLedgerSlot() const;
+  // Returns the lowest slot that the node has information about in its ledger.
```

Comment:
please use block comment before the method declaration so IDEs can pick it up

Reply:
Done the change..